third_party/2and3/tabulate.pyi

```diff
@@ -0,0 +1,16 @@
+# Stub for tabulate: https://bitbucket.org/astanin/python-tabulate
+from typing import Any, Sequence
+
+
```

Comment:
Since the stub is incomplete, please add the following at module level:

```python
def __getattr__(name: str) -> Any: ...
```

Reply:
Done.